libafl/src/corpus/testcase.rs

```diff
@@ -209,8 +220,13 @@ where
         input.wrapped_as_testcase();
         Testcase {
             input: Some(input),
+            filename: None,
+            metadata: Default::default(),
+            exec_time: None,
+            cached_len: None,
             executions,
-            ..Testcase::default()
+            fuzz_level: 0,
+            fuzzed: false,
```

Comment:
Shouldn't 0, None, and false, all be the default?

Reply:
When a type implements Drop, you can no longer use ..Default::default() because that constructs a whole object, then decomposes it, which isn't valid for Drop'd types.

Reply:
TIL